airbyte-webapp/src/components/Modal/index.stories.tsx

```diff
@@ -11,7 +11,7 @@ export default {
 } as ComponentMeta<typeof Modal>;
 
 const Template: ComponentStory<typeof Modal> = (args) => {
-  if (args.clear) {
+  if (args.cardless) {
```

Comment:
Should rename this to props.

Reply:
it looks like the convention for storybook components is `args` rather than `props`, at least for the ones in our codebase.  That sounds like a bigger chat if we want to change that.